Unable to fetch documentation archives

**Checklist (Please check before submitting)**

* [x] I reviewed the [Contributing Guide](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md).
* [x] I performed a cursory search to see if the bug report is relevant, not redundant, nor in conflict with other tickets.

**Describe the bug**
Resolve Install PDF Generation Dependencies job in build-deploy-doc.yml. The cFS Documentation and Guides workflows are failing due to the job. 

**Expected behavior**
Same behavior.

**Code snips**
```
Ign:1 http://us.archive.ubuntu.com/ubuntu bionic-updates/main amd64 libfreetype6 amd64 2.8.1-2ubuntu2.1
Ign:2 http://us.archive.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2022a-0ubuntu0.18.04
Ign:3 http://us.archive.ubuntu.com/ubuntu bionic-updates/main amd64 libtiff5 amd64 4.0.9-5ubuntu0.5
Ign:4 http://us.archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpoppler73 amd64 0.62.0-2ubuntu2.12
Err:1 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libfreetype6 amd64 2.8.1-2ubuntu2.1
  404  Not Found [IP: 91.189.91.39 80]
Err:2 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2022a-0ubuntu0.18.04
  404  Not Found [IP: 91.189.91.39 80]
Err:3 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libtiff5 amd64 4.0.9-5ubuntu0.5
  404  Not Found [IP: 91.189.91.39 80]
Err:4 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libpoppler73 amd64 0.62.0-2ubuntu2.12
  404  Not Found [IP: 91.189.91.39 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/f/freetype/libfreetype6_2.8.1-2ubuntu2.1_amd64.deb  404  Not Found [IP: 91.189.91.39 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/t/tzdata/tzdata_2022a-0ubuntu0.18.04_all.deb  404  Not Found [IP: 91.189.91.39 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/t/tiff/libtiff5_4.0.9-5ubuntu0.5_amd64.deb  404  Not Found [IP: 91.189.91.39 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/poppler/libpoppler73_0.62.0-2ubuntu2.12_amd64.deb  404  Not Found [IP: 91.189.91.39 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

**Reporter Info**
Ariel Adams, MCSG Tech

I can confirm this. After syncing my fork, I got an email from github:

```
[trholding/cFS] cFS Documentation and Guides workflow run

cFS Documentation and Guides: Some jobs were not successful

```

Root Cause seems to be this merge IC: Caelum-rc4+dev24 #587 : https://github.com/nasa/cFS/pull/587/checks

```
Run if [[ -s mission-doc-warnings.log ]]; then
  if [[ -s mission-doc-warnings.log ]]; then
    cat mission-doc-warnings.log
    exit -1
  fi
  shell: /bin/bash --noprofile --norc -e -o pipefail {0}
/home/runner/work/cFS/cFS/psp/CHANGELOG.md:1[2](https://github.com/trholding/cFS/actions/runs/3224233894/jobs/5275303108#step:14:2)0: warning: explicit link request to 'includes' could not be resolved
Error: Process completed with exit code 2[5](https://github.com/trholding/cFS/actions/runs/3224233894/jobs/5275303108#step:14:5)5.
```

--- a/.github/workflows/build-deploy-doc.yml
+++ b/.github/workflows/build-deploy-doc.yml
@@ -103,7 +103,7 @@ jobs:
         run: make prep
 
       - name: Install Doxygen Dependencies
-        run: sudo apt-get install doxygen graphviz -y
+        run: sudo apt-get update && sudo apt-get install doxygen graphviz -y
 
       - name: Install PDF Generation Dependencies
         if: ${{ inputs.buildpdf == true }}

